.travis.yml

```diff
@@ -1,40 +1,40 @@
 language: python
 python:
-  - '2.7'
-  - '3.4'
-  - '3.5'
-  - '3.6'
+- '2.7'
```

Comment:
Was this intentional? I think the old formatting looked correct.

Reply:
Hmm, I think this has happened to me before when running a travis-cli command on an existing .travis.yml file (in this case, I ran `travis setup pypi`). For some reason it reformats the whole file? In any case I agree that the previous formatting was nicer, so I'll change it back.